capa/features/extractors/elf.py

```diff
@@ -160,24 +159,28 @@ def _parse(self):
             raise CorruptElfFile(f"invalid ei_class: 0x{ei_class:02x}")
 
         if ei_data == 1:
-            self.endian = "<"
+            self.endian = "little"
         elif ei_data == 2:
-            self.endian = ">"
+            self.endian = "big"
         else:
             raise CorruptElfFile(f"not an ELF file: invalid ei_data: 0x{ei_data:02x}")
 
         if self.bitness == 32:
-            e_phoff, e_shoff = struct.unpack_from(self.endian + "II", self.file_header, 0x1C)
-            self.e_phentsize, self.e_phnum = struct.unpack_from(self.endian + "HH", self.file_header, 0x2A)
-            self.e_shentsize, self.e_shnum, self.e_shstrndx = struct.unpack_from(
-                self.endian + "HHH", self.file_header, 0x2E
-            )
+            e_phoff = int.from_bytes(self.file_header[0x1C:0x20], byteorder=self.endian, signed=False)
+            e_shoff = int.from_bytes(self.file_header[0x20:0x24], byteorder=self.endian, signed=False)
```

Comment:
i find this region to be very noisy and hard to read. is there some way to refactor the repeated leaflets, for example?